container/rockylinux9-cuda12/Dockerfile

```diff
@@ -7,39 +7,38 @@ RUN dnf -y install \
     redhat-rpm-config \
     which \
     xz \
-    zlib-devel
-
+    zlib-devel \
 # Needed to build `path_tracer`, `optix/ex03_window` example
-RUN dnf -y install \
     cmake \
     fontconfig-devel \
     libX11-devel  \
     libXcursor-devel \
     libXi-devel \
-    libXrandr-devel
+    libXrandr-devel && \
+    dnf clean all
 
 # Get LLVM 7 & libffi.so.6
 WORKDIR /data/llvm7
-RUN curl -sSf -L -O https://dl.fedoraproject.org/pub/epel/9/Everything/x86_64/Packages/l/libffi3.1-3.1-36.el9.x86_64.rpm
-RUN curl -sSf -L -O https://dl.fedoraproject.org/pub/epel/8/Everything/x86_64/Packages/l/llvm7.0-7.0.1-7.el8.x86_64.rpm
-RUN curl -sSf -L -O https://dl.fedoraproject.org/pub/epel/8/Everything/x86_64/Packages/l/llvm7.0-devel-7.0.1-7.el8.x86_64.rpm
-RUN curl -sSf -L -O https://dl.fedoraproject.org/pub/epel/8/Everything/x86_64/Packages/l/llvm7.0-libs-7.0.1-7.el8.x86_64.rpm
-RUN curl -sSf -L -O https://dl.fedoraproject.org/pub/epel/8/Everything/x86_64/Packages/l/llvm7.0-static-7.0.1-7.el8.x86_64.rpm
-RUN dnf -y install ./*.rpm
-RUN ln -s /usr/bin/llvm-config-7-64 /usr/bin/llvm-config
+RUN curl -sSf -L -O https://dl.fedoraproject.org/pub/epel/9/Everything/x86_64/Packages/l/libffi3.1-3.1-36.el9.x86_64.rpm && \
```

Comment:
Merging curl RUNs are fine since I think no many people would like to modify it.